Yoast/javascript
# Add Russian morphology support

## Summary


This PR can be summarized in the following changelog entry:

* [yoastseo] Adds a stemmer for the Russian language.
* [Yoast SEO Premium] Adds word form support for Russian.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* In Russian Premium: use different forms of Russian words in keyphrase and in text/title/meta description and check that everything gets recognized. E.g., "глупость", "глупости", "глупостью", "глупостями", "глупостях", "глупый", "глупейше".

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes #
